### PR TITLE
Add What's new entry for NDData

### PR DESCRIPTION
**Pull request by mwcraig**
@MSeifert04 -- can you please take a look at this?

### CODE REVIEW

File: docs/whatsnew/1.2.rst [hunk 1]
```diff
@@ -60,6 +60,17 @@ NDData improvements
 -------------------
 
 
+Arithmetic and uncertainty handling have been enhanced for astropy 1.2, and a
+new class, `astropy.nddata.NDDataRef`, has been added that should be the
+starting point for new users of `nddata`. One can now do arithmetic between an
```
Comment (MSeifert04) [inline docs/whatsnew/1.2.rst]: :ref:"astropy_nddata" (backticks instead of the quotes.

File: docs/whatsnew/1.2.rst [hunk 2]
```diff
@@ -60,6 +60,17 @@ NDData improvements
...
+`astropy.nddata.NDDataRef` object and a scalar, an `astropy.units.Quantity` or
+another `astropy.nddata.NDDataRef` object (or any class that uses the
+`astropy.nddata.NDArithmetic` mixin). Limited support for propagating
```
Comment (MSeifert04) [inline docs/whatsnew/1.2.rst]: NDArithmeticMixin.

**Comment by MSeifert04** [comment]
@mwcraig Thank you. I had totally forgotten about this (after I had some problems doing a PR against astrofrogs branch). Apart from the two sphinx failures this looks good.  Do you think any examples should be included as well? Unfortunatly I'm gone this weekend so I could only provide some at monday/tuesday.

**Comment by mwcraig** [comment]
Comments addressed, thanks @MSeifert04

**Comment by mwcraig** [comment]
@MSeifert04 -- I think examples are better left to the docs for the subpackage.

**Pull request merged**